src/setupWorker/glossary.ts

```diff
@@ -37,6 +37,11 @@ export type ServiceWorkerInstanceTuple = [
   ServiceWorkerRegistration,
 ]
 
+export type ServiceWorkerMatcher = (
+  scriptUrl: string,
+  absoluteWorkerUrl: string,
```

Comment:
Diving more into the implementation, I think the call signature of this function is confusing as well. `scriptUrl` is a worker script URL of each registered worker. `absoluteWorkerUrl` is the URL of the MSW's service worker (absolute version of the URL you provided as `start({ serviceWorker: { url } })`, given to you for convenience. 

I'm thinking of something like this:

```ts
export type FindWorker = (scriptUrl: string, mockServiceWorkerUrl: string) => boolean
```

Reply:
I agree, this makes sense to me.